src/devices_models/devices/AC_branches.jl

```diff
@@ -104,13 +92,8 @@ function branch_rate_bounds!(psi_container::PSIContainer,
         push!(constraint_data,
               DeviceRange(name, limit_values, services_ub, Vector{Symbol}()))
     end
-    set_variable_bounds(psi_container,
-                        constraint_data,
-                        Symbol("FpFT_$(B)"))
-
-    set_variable_bounds(psi_container,
-                        constraint_data,
-                        Symbol("FpTF_$(B)"))
+    set_variable_bounds!(psi_container, constraint_data, FLOW_REAL_POWER_FROM_TO, B)
```

Comment:
`FLOW_REAL_POWER_FROM_TO` could be a constraint or variable name. This is an example of such a case.